desktop/apps/auction/components/layout/Banner.js

```diff
@@ -56,15 +58,16 @@ function Banner (props) {
           return (
             <div className={b('closed')}>
               <div>
-                Auction Closed
+                {type} Closed
               </div>
             </div>
           )
-        } else {
+        } else if (hasEndTime) {
```

Comment:
Ahh @damassi I missed this before, but live sales won't have an end time, so this is causing a bug now where the clock doesn't show up for live auctions that are opening for live bidding. I think this logic should be `if (isAuction || hasEndTime)`.